Readme.md

```diff
@@ -1,46 +1,74 @@
-# Setup Steps
+# Working with Docs
 
-- If using docker see [Docker](#docker)
+- See [Docker](#docker) steps if you're using **Docker container**. See [Virtual Environment](#virtual-environment) if you're using **Python venv**.
 
-- If python virtual environment is already created for your local docs directory see [Usage](#usage) steps else start with [Installation](#installation).
-
-- Checkout the docs directory and execute the below commands.
+- Checkout this repo and execute the below commands.
 
 ## Docker
 
 ```console
 > docker build . -t sphinx:latest
 > docker run --rm -it -h sphinx -v $(pwd):/mnt/host/docs -w /mnt/host/docs -p 8000:8000 sphinx
-> sphinx-autobuild ./source/ ./build/html/ --port 8000 --host 0.0.0.0
 ```
 
-## Installation
+You can now begin on documentation and following the [build steps](#build-steps) guide.
+
+---
+
+## Virtual Environment
+
+If python virtual environment is **not** created for your local docs directory start with [Installation](#installation) steps for Ubuntu, else you can activate the virtual environment using [Activation and Deactivation](#activation-and-deactivation) steps.
+
+### Installation
+
+- The below steps require Python3.8 to be installed. To install using APT manager use the below command.
+
+```console
+> sudo apt install python3.8
+```
+
+- After verifying Python3.8 execute the below commands to create the virtual environment.
 
 ```console
 > python3 -m venv venv
 > source venv/bin/activate
 > pip3 install -r requirements.txt
 ```
 
-## Usage
+### Activation and Deactivation
+
+To activate the virtual environment created in your repo follow the below step. **Ignore** if already activated.
 
 ```console
 > source venv/bin/activate
-> make html -b spelling -b linkcheck
-> sphinx-autobuild ./source/ ./build/html/ --port 8000 --host 0.0.0.0
+
+> # Exit/Deactivate the entered virtual environment run 
```

Comment:
Mention the activation and deactivation steps separately with a description before each

Reply:
Done